README.md

```diff
@@ -273,11 +273,13 @@ spec:
 
 #### ServiceBindingRequestStatus
 
-`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
+`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
 
-| Field | Description |
-|-------|-------------|
-| BindingStatus | The binding status of Service Binding Request |
-| Secret | The name of the intermediate secret |
+```
+status:
+  bindingStatus: success
+  secret: binding-request 
```

Comment:
In order to support multiple ServiceBindingRequests per namespace, is a unique name assigned to each binding-request intermediate secret?

Reply:
Yes, this is the name of the unique secret that gets created for the specific request. 

( need to check, there needs to be a fix to ensure a unique name is picked while creating the intermediate secret..)